examples/android_local_test/src/test/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+android_local_test(
+    name = "main_activity_test",
```

Comment:
General file comment: why isn't the BUILD file next to MainActivityTest? If it was, and you called the target "MainActivityTest", you wouldn't need the test_class attribute. I don't have a strong preference, but I do think we recommend this practice at least for java tests

Reply:
Hmm, I wanted to mirror the structure in src/main. I'll leave it as it is for now.